implementations/elixir/ockam/ockam/test/ockam/secure_channel_test.exs

```diff
@@ -76,7 +83,7 @@ defmodule Ockam.SecureChannel.Tests do
 
       Router.route(message)
 
-      if rem(i, 2) == 1 do
+      if rem(i, 2) == 0 do
```

Comment:
I mildly shocked to see that there's no `is_even` function in Elixir? Would it make sense to make it a utility function somewhere?

Reply:
there is one macro on Integer,  didn't check before writting the test